docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -144,34 +144,45 @@ all labeled and additional information will be included below.
 
 
 .. list-table:: Outputs from Pregnancy Decision Tree
-  :widths: 3 15 15
+  :widths: 3 15 15 15
   :header-rows: 1
 
   * - Input
     - Data Source 
-    - Notes
+    - Overall stratification
+    - Use case 
   * - ANC attendance
```

Comment:
This is not specific enough -- we need to specify how to measure ANC attendance. For example, we could say "ANC attendance (attended or did not attend)".

Reply:
Agree that we should make it more specific, but should I request a counts table rather than a state table? I thought that maybe specifying stratifications implied counts but maybe I should be more explicit?

Reply:
Hmm, yeah, this is related to my comment below. I'm not sure whether it's better to have the full population table or whether we want only some specific aggregate outputs from observers. Perhaps we can discuss at the MNCNH RT meeting tomorrow.

Reply:
```suggestion
  * - ANC attendance (true if attended ANC, false if not)
```